### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Varlaisvea**
- autolathes can now actually be deconstructed and don't cause runtimes (and also don't eat your crowbars)
- forensic techs actually get detective gear now (ID, PDA, CSI kit)
- accessories don't have that weird double period
- right foot
- stacks now list "there is 1" instead of "there are 1" when there's only one
- ghosts can actually hang up on communicators, it still runtimes but it did that before and the runtime is an observer bug instead of a communicator bug someone else can fix that
- the hos actually gets an egun now, they were always meant to have Something but their fancy custom gun was removed when we swapped to cynosure and never readded
- you can no longer generate infinite power cells with autolathes (sorry)

### CODE REVIEW

**Comment by Varlaisvea** [comment]
fixes #8948
fixes #8916 
fixes #8901 
fixes #8893 
fixes #8879 
fixes #8130 (kinda, you have to deconstruct but it's still a fix)
fixes a bunch of bugs that i didn't add to the tracker

File: code/game/machinery/autolathe.dm [hunk 1]
```diff
@@ -155,10 +155,12 @@
 		return
 
 	if(panel_open)
-		//Don't eat multitools or wirecutters used on an open lathe.
+		//Don't eat things when the lathe is open. No more accidentally lathing your jaws of life.
 		if(O.is_multitool() || O.is_wirecutter())
 			wires.Interact(user)
 			return
+		else
+			return 0
```
Comment (Atermonera) [inline code/game/machinery/autolathe.dm:163]:
```suggestion
			return TRUE
		return FALSE
```

File: code/game/machinery/autolathe.dm [hunk 2]
```diff
@@ -362,14 +364,17 @@
...
-	for(var/mat in stored_material)
-		var/datum/material/M = get_material_by_name(mat)
-		if(!istype(M))
-			continue
-		var/obj/item/stack/material/S = new M.stack_type(get_turf(src))
-		if(stored_material[mat] > S.perunit)
-			S.amount = round(stored_material[mat] / S.perunit)
-		else
-			qdel(S)
+	if(stored_material)
```
Comment (Atermonera) [inline code/game/machinery/autolathe.dm:367]:
```suggestion
	if(LAZYLEN(stored_materials))
```

File: code/game/objects/items/devices/flashlight.dm [hunk 3]
```diff
@@ -234,6 +236,9 @@
 	else
 		..()
 
+/obj/item/flashlight/empty //For autolathes so you can't duplicate infinite cells
+	starts_with_cell = 0
```
Comment (Atermonera) [inline code/game/objects/items/devices/flashlight.dm:240]: FALSE

File: code/game/objects/items/devices/suit_cooling.dm [hunk 4]
```diff
@@ -24,6 +24,7 @@
 	var/max_cooling = 15				// in degrees per second - probably don't need to mess with heat capacity here
 	var/charge_consumption = 3			// charge per second at max_cooling
 	var/thermostat = T20C
+	var/starts_with_cell = 1
```
Comment (Atermonera) [inline code/game/objects/items/devices/suit_cooling.dm:27]: TRUE/FALSE

File: html/changelogs/Bugfixes - Varlaisvea.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name.  
+author: Varlaisvea
```
Comment (Atermonera) [inline html/changelogs/Bugfixes - Varlaisvea.yml:25]: fyi - manual changelog updates have been all but abandoned, and the autochangelog has been broken for >1 year (and I can't be arsed to fix it)

**Comment by Varlaisvea** [comment]
also adds missing icons for retro carpet (red), icons already existed but were incorrectly labelled in stacks.dmi

File: html/changelogs/Bugfixes - Varlaisvea.yml (hunk 5 above)
Comment (Varlaisvea) [reply html/changelogs/Bugfixes - Varlaisvea.yml:25]: oh thank god changelogs are hell

File: code/game/machinery/autolathe.dm [hunk 6]
```diff
@@ -362,14 +363,17 @@
...
+			if(stored_material[mat] == 0) //Maybe don't try and make null mats...
+				continue
+			var/obj/item/stack/material/S = new M.stack_type(get_turf(src))
+			if(stored_material[mat] >= S.perunit)
+				S.amount = round(stored_material[mat] / S.perunit)
+			else
+				qdel(S) //Prevents stacks smaller than 1
 	..()
-	return 1
+	return 1
```
Comment (Atermonera) [inline code/game/machinery/autolathe.dm:379]:
```suggestion
	return ..()
```

**Pull request merged**